apps/api/src/modules/health/health.controller.ts

```
@@ -0,0 +1,70 @@
+import { PrismaClient } from '@saas/database'
+import type { FastifyReply, FastifyRequest } from 'fastify'
+import { Redis } from 'ioredis'
+import { inject, injectable } from 'tsyringe'
+import { logger } from '../../shared/utils/logger.js'
```

ESLint

[error] 'logger' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)